### ISSUE TITLE
Improve host connection/disconnection alert configuration

### ISSUE THREAD

**Issue by lucaderi** [ntop/ntopng#2173]
In 
![screen shot 2018-11-21 at 13 15 27](https://user-images.githubusercontent.com/4493366/48840644-8ad32600-ed8f-11e8-8c07-c2b8cfc4f09b.png)

it is not possible to specify for what hosts the alert should be triggered. This practice generate alerts also for hosts that are not supposed to generate alerts. Said that this ticket must be implemented as soon as #1854 will be implemented, it would be nice to have the ability to specify for what hosts a specific alert should (not) be triggered.

**Comment by emanuele-f** [comment]
This is what I propose:

- Turn the global preference into a "*New* devices connection/disconnection.."
- Add an option on the host pools configuration page, disabled by default, to trigger these alerts on the specific host pool

**Comment by lucaderi** [comment]
The first option does not work, the second will be hard to implement as people are used to use host preferences. As this issue depends on  #1854 let's wait until that is implemented

**Comment by lucaderi** [comment]
4.2 has a plugin for detecting these issues. So it is implemented now
